### ISSUE TITLE
Limitation of each API call

### ISSUE THREAD

**Issue by Henry922** [datadudes/salesforce2hadoop#20]
Hi just wondering is there a limitation on number of records being pulled from salesforce to hadoop using sf2hadoop ? my understanding REST API do have limitations on number calls, records..etc
And does this impact to our initial load ?

Thanks
Henry

**Comment by DonDebonair** [comment]
You are correct that the Salesforce REST API only allows you to pull in a certain number of records per request. sf2hadoop works around that by doing multiple requests for the initial load, of 2000 records each, until all records have been stored.
